backend/src/api/mutation.rs

```diff
@@ -58,6 +60,22 @@ impl Mutation {
         Realm::remove(id, context).await
     }
 
+    /// Adds a title block to a realm.
+    ///
+    /// The new block will be inserted at the given index,
+    /// i.e. it will be at that position after the insert.
+    /// Or, if you prefer to think about it this way:
+    /// It will be inserted before the block that currently sits
+    /// at that index.
```

Comment:
Mh... doc duplication. I wonder if we can somehow improve that. Maybe just describe it for dunno "add_text" and then refer to that in all other docs?

Reply:
:shrug:

This raises interesting questions about how to refer to other mutations, though. Links are apparently supported by the GraphQL doc format (read: GraphiQL displays them), but what do we link to? Or do we treat the comments as rust docs and just write `[add_title_block]`? That shows up verbatim in the GraphQL schema, though, and is the wrong casing in that context.

Reply:
I would prefer GraphiQL links. And well, just pick any of the content blocks to serve as "explainer" and make everything else link to it? I'd say

Reply:
Ah ok. Then just refer to something else by name, I'd say. Not aaas convenient, but people will just navigate there then.